### ISSUE TITLE
After posting a learn, a link/button should allow easy sharing to Twitter

### ISSUE THREAD

**Issue by louiseswift** [no-mayo/today-i-learned#3]

**Comment by mpettaw2** [comment]
Hello, 
I hope you are doing well? I had a question, do you want your entire "data" object to be formatted and sent to twitter you just a single property?

**Comment by louiseswift** [comment]
Hello!

Good question. I think for this particular issue/feature, we'd like a link/button to appear (after a new learn is successfully added) that suggests a tweet like: _"TIL: [new learn detail] #TIL /cc @no_mayo_digital"_ - tagging us so that we can reply with a :star2:!

Twitter's [Web Intents](https://dev.twitter.com/web/intents) (specifically the [Tweet Web Intent](https://dev.twitter.com/web/tweet-button/web-intent)) should be fine for this - and I just found a tool that looks useful for generating the correct href for a click-to-tweet link/button: http://tech.cymi.org/tweet-intents

Let us know if you have any further questions about this issue! We created this project specifically to offer 'beginner'-tagged issues for newer developers, mostly ones who might be participating in one of our London-based mentoring sessions that are starting soon - but if you're happy to tackle the issue remotely, we'd be happy to accept your contribution too!
